### PR TITLE
Issue #1147: add strings for Turbo Mode onboarding

### PR DESCRIPTION
**Pull request by Baron-Severin**

### CODE REVIEW

File: app/src/main/res/values/strings.xml [hunk 1]
```diff
@@ -181,10 +181,13 @@ be temporary, and you can try again later.</li>
     <string name="turbo_mode_disabled_toast">Turbo Mode disabled</string>
     <string name="onboarding_turbo_mode_title">Turbo Mode helps you browse faster</string>
     <string name="onboarding_turbo_mode_body">Turbo Mode automatically blocks trackers and ads to help get you where you\'re going faster. If there\'s a speed bump – like a site that isn\'t behaving the way you expect – just turn Turbo Mode off.</string>
+    <string name="onboarding_turbo_mode_body2">Turbo Mode automatically blocks trackers and ads to help get you where you’re going faster. If there’s a speed bump – like a site that isn’t behaving the way you expect – just turn Turbo Mode off.</string>
```
Comment (mcomella) [inline app/src/main/res/values/strings.xml]: Did you try building this? I think you need to escape the apostrophes.

Comment (mcomella) [inline app/src/main/res/values/strings.xml]: How is this different from the string that's above it? The strings should be named to indicate *where* in the UI they're used.

Are the old strings no longer used? In that case you should remove them.

Comment (mcomella) [reply app/src/main/res/values/strings.xml]: Then again, if these strings are identical and used in the same context (just with a redesigned layout), maybe we should just use the first string.

Comment (Baron-Severin) [reply app/src/main/res/values/strings.xml]: Yeah, I was surprised as well, but it looks like the curved apostrophes don't need to be escaped.

Comment (Baron-Severin) [reply app/src/main/res/values/strings.xml]: The only difference is using curved apostrophes instead of normal ones.

Comment (Baron-Severin) [reply app/src/main/res/values/strings.xml]: To be clear, I did build to double check.

Comment (Baron-Severin) [reply app/src/main/res/values/strings.xml]: I can remove the old strings when I merge the layout changes, but until then we need them or compilation will break.

Comment (mcomella) [reply app/src/main/res/values/strings.xml]: @Baron-Severin I feel like delphine has previously said we shouldn't have curly apostrophes in strings, but I'm checking right now.

File: app/src/main/res/values/strings.xml [hunk 2]
```diff
@@ -181,10 +181,13 @@ be temporary, and you can try again later.</li>
...
     <!-- Button text for keeping Turbo Mode enabled in onboarding -->
     <string name="button_turbo_mode_keep_enabled">Keep enabled</string>
+    <string name="button_turbo_mode_keep_enabled2">Keep Turbo Mode enabled</string>
     <!-- Button text for turning off Turbo Mode in onboarding -->
     <string name="button_turbo_mode_turn_off">Turn off</string>
+    <string name="button_turbo_mode_turn_off2">Turbo Turbo Mode off</string>
```
Comment (dnarcese) [inline app/src/main/res/values/strings.xml]: Should this say '_Turn_ Turbo Mode off'?

File: app/src/main/res/values/strings.xml (hunk 1 above)
Comment (mcomella) [reply app/src/main/res/values/strings.xml]: Why are these strings merging independently of the code? In general, it's better to land strings with the code so localizers can see where their strings will be used.

Comment (mcomella) [reply app/src/main/res/values/strings.xml]: Sounds like it was the opposite: everything should be curly. lgtm.

File: app/src/main/res/values/strings.xml (hunk 2 above)
Comment (Baron-Severin) [reply app/src/main/res/values/strings.xml]: Good catch

**Comment by Baron-Severin** [comment]
Closed, adding the strings in along with the PR in which they're used.

**Pull request closed without merging**